New Slovak translation - celestial_objects.xml

MilanSL opened the pull request:
Added new strings from original xml and changed translation to Slovak language which is missing for all Sky Map app.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `gradlew app:connectedGmsDebugAndroidTest` to make sure you didn't break anything (with an emulator running or phone connected)

- [x] If you have multiple commits please combine them into one commit by squashing them.

*Note* you may see the Cirrus CI system failing on your PR. It's pretty flaky right now so don't
be alarmed.

jaydeetay commented:
Thanks! Looks like this is failing to build because some of the new strings you've added already exist in values.xml ?

See: https://github.com/sky-map-team/stardroid/pull/541/checks?check_run_id=37781498502

MilanSL commented:
Hi, strings added to celestiaI_objects.xml were deleted from strings.xml. For example, I deleted Messier objects from one xml and did copy them to another one . Therefore I do not understand why this error appeares. Maybe the check for celestial_objects.xml was done before I deleted those strings from strings.xml file.
